src/CustomerBundle/Block/RecentCustomersBlockService.php

```diff
@@ -25,7 +25,7 @@
 /**
  * @author Hugo Briand <briand@ekino.com>
  */
-class RecentCustomersBlockService extends BaseBlockService
+class RecentCustomersBlockService extends AbstractBlockService
```

Comment:
this needs to extend from: `AbstractAdminBlockService`, as it is for the backend